### PR TITLE
Manage static dns entries front-end changes [WIP]

### PR DESCRIPTION
**Pull request by rivasj**

### CODE REVIEW

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/trafficcontrol-PR/2152/
Test PASSed.

File: traffic_portal/app/src/common/modules/table/deliveryServiceStaticDnsEntries/TableDeliveryServiceStaticDnsEntriesController.js [hunk 1]
```diff
@@ -27,6 +27,10 @@ var TableDeliveryServiceStaticDnsEntriesController = function(deliveryService, s
 		$state.reload(); // reloads all the resolves for the view
 	};
 
+    $scope.editStaticDnsEntry = function(dsId, staticDnsEntryId) {
+        locationUtils.navigateToPath('/delivery-services/' + dsId + '/static-dns-entry/' + staticDnsEntryId);
```
Comment (mitchell852) [inline traffic_portal/app/src/common/modules/table/deliveryServiceStaticDnsEntries/TableDeliveryServiceStaticDnsEntriesController.js]: you should make this plural to conform with the rest of TP's routes. so like this

+        locationUtils.navigateToPath('/delivery-services/' + dsId + '/static-dns-entries/' + staticDnsEntryId);

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/trafficcontrol-PR/2153/
Test PASSed.

File: traffic_portal/app/src/modules/private/deliveryServices/staticDnsEntries/list/index.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+				views: {
+					deliveryServicesStaticDnsEntriesContent: {
+						templateUrl: 'common/modules/table/deliveryServiceStaticDnsEntries/table.deliveryServiceStaticDnsEntries.tpl.html',
+						controller: 'TableDeliveryServiceStaticDnsEntriesController',
+						resolve: {
+							deliveryService: function($stateParams, deliveryServiceService) {
+								return deliveryServiceService.getDeliveryService($stateParams.deliveryServiceId);
+							},
+							staticDnsEntries: function($stateParams, deliveryServiceStaticDnsEntryService) {
+								return deliveryServiceStaticDnsEntryService.getStaticDnsEntries({ deliveryserviceId: $stateParams.deliveryServiceId });
```
Comment (mitchell852) [inline traffic_portal/app/src/modules/private/deliveryServices/staticDnsEntries/list/index.js]: so here you refer to `deliveryServiceStaticDnsEntryService` but in traffic_portal/app/src/common/api/index.js you have named it `staticDnsEntryService`

**Pull request closed without merging**